src/integrationTest/kotlin/com/sourcegraph/cody/util/CustomJunitClassRunner.kt

```diff
@@ -0,0 +1,13 @@
+package com.sourcegraph.cody.util
+
+import org.junit.runners.BlockJUnit4ClassRunner
+import org.junit.runners.model.FrameworkMethod
+
+class CustomJunitClassRunner(klass: Class<*>?) : BlockJUnit4ClassRunner(klass) {
```

Comment:
Maybe also something with `Repeatable` or `Repeat` in name? `Custom` doesn't bring too much context

Reply:
This class doesn't really repeat anything or make anything repeatable. I don't mind renaming it, but I can't think of any good name.